app/src/main/java/com/github/se/wanderpals/model/viewmodel/AdminViewModel.kt

```diff
@@ -22,7 +22,8 @@ open class AdminViewModel(
   open var listOfUsers = MutableStateFlow(emptyList<User>())
   // delete a user from the database
   open fun deleteUser(userId: String) {
```

Comment:
Great update to listOfUsers to not require pulling the database to get an updated listOfUsers